src/ios/CDVPurchasesPlugin.m

```diff
@@ -93,73 +86,61 @@ - (void)purchasePackage:(CDVInvokedUrlCommand *)command
                            completionBlock:[self getResponseCompletionBlock:command]];
 }
 
-- (void)restoreTransactions:(CDVInvokedUrlCommand *)command
-{
+- (void)restoreTransactions:(CDVInvokedUrlCommand *)command {
     [RCCommonFunctionality restoreTransactionsWithCompletionBlock:[self getResponseCompletionBlock:command]];
 }
 
-- (void)getAppUserID:(CDVInvokedUrlCommand *)command
-{
+- (void)syncPurchases:(CDVInvokedUrlCommand *)command {
+    [RCCommonFunctionality syncPurchasesWithCompletionBlock:[self getResponseCompletionBlock:command]];
+}
+
+- (void)getAppUserID:(CDVInvokedUrlCommand *)command {
     CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:[RCCommonFunctionality appUserID]];
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
-- (void)createAlias:(CDVInvokedUrlCommand *)command
-{
+- (void)createAlias:(CDVInvokedUrlCommand *)command {
     NSString *newAppUserID = [command argumentAtIndex:0];
 
     [RCCommonFunctionality createAlias:newAppUserID completionBlock:[self getResponseCompletionBlock:command]];
 }
 
-- (void)identify:(CDVInvokedUrlCommand *)command
-{
+- (void)identify:(CDVInvokedUrlCommand *)command {
```

Comment:
formatting here was inconsistent, so I just moved the file to this format, which we use in the native sdk